src/sagas/wallet/sagas.js

```diff
@@ -16,6 +17,39 @@ import type { SendMoneyAction } from '../../actions/wallet';
 import type { WalletType } from '../../types/Wallet';
 import ServiceContainer from '../../services/container';
 import { NoWalletServiceError } from '../../global/errors/services';
+import defaultDB from '../../services/database';
+import type { WalletType as DBWallet } from '../../services/database/schemata';
+import { convertFromDatabase, convertToDatabase } from '../../utils/mapping/wallet';
+import { resolveWallet } from '../../utils/wallet';
+
+/**
+ * @desc Update Eth wallet balance to realm.
+ * @param {WalletType[]} walletsArray Array of wallets to save in to Realm
+ * @param {String} amount Amount to discount from the wallet
+ * @param {String} currency Currency of the wallet to update
+ * @returns {void}
+ */
+export function* updateWalletToDb(walletsArray: WalletType[], amount: string, currency: string): Generator<*, *, *> {
+  console.log('Wallet Params --> ', walletsArray);
```

Comment:
Please remove console.logs from here.

Reply:
Sorry about this, totally forgot to remove them all